moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/collision_check_thread.h

```diff
@@ -39,17 +39,17 @@
 
 #pragma once
 
-#include <jog_arm/jog_arm_data.h>
-#include <jog_arm/low_pass_filter.h>
+#include <moveit_jog_arm/jog_arm_data.h>
```

Comment:
You are not going to install the headers, do you?
```suggestion
#include "jog_arm_data.h"
```

Reply:
I take it that all of these header files need this change, not just this one?

Reply:
I thought it was a ROS convention to put header files in include/${PACKAGE_NAME} so they won't clash with other packages if header files ever do need to be installed. Will ignore this suggestion for now.

Reply:
(I appreciate learning the difference between "" and <> though)